src/covid_model_seiir_pipeline/postprocessing/task/postprocess.py

```diff
@@ -0,0 +1,184 @@
+from argparse import ArgumentParser, Namespace
```

Comment:
I'm surprised `git mv` didn't call this a rename.  Same as old postprocess.py task with added aggregation and splicing features.